webapp/src/features/adminNotificationTab/AdminNotificationTab.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react'
+import AdminTable from '../../components/AdminTable'
+import { IoChevronBackCircleOutline, IoRefresh } from 'react-icons/io5'
+import { AiOutlinePlusCircle } from 'react-icons/ai'
+import { confirmAlert } from 'react-confirm-alert'
+import { Options } from '../../components/AdminDeletionOptions'
+import { selectLoading } from '../../generalSlices/rsuSlice'
+import {
+  selectTableData,
+  getUserNotifications,
+  deleteNotifications,
+  updateTitle,
+  setActiveDiv,
+  setEditNotificationRowData,
+} from './adminNotificationTabSlice'
+import { useSelector, useDispatch } from 'react-redux'
+
+import '../adminRsuTab/Admin.css'
+import { AnyAction, ThunkDispatch } from '@reduxjs/toolkit'
+import { RootState } from '../../store'
+import { Action } from '@material-table/core'
+import { Route, Routes, useLocation, useNavigate } from 'react-router-dom'
+import { NotFound } from '../../pages/404'
+import AdminEditNotification from '../adminEditNotification/AdminEditNotification'
+import AdminAddNotification from '../adminAddNotification/AdminAddNotification'
+import { AdminEmailNotification } from '../../types/Notifications'
+
+const getTitle = (activeTab: string) => {
+  if (activeTab === undefined) {
+    return 'CV Manager Email Notifications'
+  } else if (activeTab === 'editNotification') {
+    return 'Edit Email Notifications'
+  } else if (activeTab === 'addNotification') {
+    return 'Add Email Notifications'
+  }
+  return 'Unknown'
+}
+
+const AdminNotificationTab = () => {
```

Comment:
I am conflicted. Using the table is a great way to dynamically support other email notifications that may be added in the future without having to redesign a menu. However, I feel like the way it is currently presented makes it unclear that this is specifically a menu for the user and no one else. This may be more confusing to an administrator since it is on the admin tab.

On another note, will a general user ever be able to access this if they are not an admin? I don't think so but I haven't personally tried this.

Reply:
Currently, users can't access this menu unless they are an admin. Would it be better to pull this into its own tab? That way users could configure their notifications without the admin role. Regardless, I believe some renaming might make the purpose of this tab clearer - do you have any ideas on how clarity could be improved?

Reply:
It might make sense to do that, yeah. I am more in favor of a new tab for user settings. Not sure if I have a preference on what to name the tab. "User Settings" maybe?

Reply:
Added a new 'User Settings' tab with the notification logic